mlrun/feature_store/retrieval/spark_merger.py

```diff
@@ -57,30 +57,41 @@ def _generate_vector(
             feature_set = feature_set_objects[name]
             feature_sets.append(feature_set)
             column_names = [name for name, alias in columns]
-            target = get_offline_target(feature_set)
-            if not target:
-                raise mlrun.errors.MLRunInvalidArgumentError(
-                    f"Feature set {name} does not have offline targets"
-                )
+
+            if feature_set.spec.passthrough:
+                if not self.spec.source:
+                    raise mlrun.errors.MLRunNotFoundError(
+                        f"passthrough feature set {name} with no source bounded"
```

Comment:
```suggestion
                        f"passthrough feature set {name} with no source"
```
If you really want to keep it though, the word you're looking for is "bound".

Reply:
fixed